### ISSUE TITLE
Accordions fail Target Size (Minimum) (Level AA) when content starts with a link

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#8505]
# Problem 

In Siteimprove: Interactive element does not meet minimum size nor spacing for Accordions when content starts with a link.

See report at https://my2.siteimprove.com/Dashboard/32195086775/Custom/Dashboard/Index and http://my2.siteimprove.com/Inspector/1001225/A11Y/Page?pageId=47390456998&impmd=264ED37B65FA6186129334A508391970#/sia-r113/failed/a01DN3haQ1Q=;NTUwOGExN2Y=. 

- https://alfa.siteimprove.com/rules/sia-r113
- https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum

# How to reproduce

View http://careers.uiowa.edu/post-grad-data and run Siteimprove Accessibility Checker

# Proposed solution

In UIDS, increase accordion content padding from `1rem` which will give accordions starting with a link enough space to meet the minimum target size. 
- Update padding https://github.com/uiowa/uids/blob/4.x/src/scss/components/accordion.scss#L75.  Make sure summary padding on the left matches if it is changed for the content area.
- Add last-child rule for `<p>` tags in accordion to remove margin.
- Test locally in chrome with https://www.siteimprove.com/integrations/browser-extensions/
- Update uids_base with new UIDS hash at https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/package.json#L6

**Comment by briand44** [comment]
Grooming conversation... Increase padding for all accordion content, no logic needed to check if a link is the first thing present in the content. Ben to add some additional info about where to update in SiteNow to use this once resolved in UIDS. Concerns about title and content not lining up. Discussed limiting to top and bottom padding. Concerns around <p> and having too much space. We could try adjusting padding to something less than 2rem and see if it passes or apply to first or last child to keep padding even between top and bottom. Should be able to test with Siteimprove chrome extension.

**Comment by briand44** [comment]
Opening this back up as I am seeing similar SiteImprove errors still.

**Comment by briand44** [comment]
Here are a couple examples I am seeing in SiteImprove currently:

https://my2.siteimprove.com/Inspector/998442/A11y/Page?pageId=56128097800&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r113/failed/NklwQjlPYlA=;M2I2YmFiOTU=

https://my2.siteimprove.com/Inspector/988894/A11y/Page?pageId=24665074256&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r113/failed/eFM2OWpKWWw=;ZTlkYzdhOGY=

Also able to see these issues on these pages using the SiteImprove browser plugin:

https://engineering.uiowa.edu/bme/undergraduate

https://education.uiowa.edu/international-day

**Comment by bspeare** [comment]
When we solved for this in https://github.com/uiowa/uids/pull/974, we were only looking at links in `<p>` tags.  These new results are for `<ul>`, and often times the WYSIWYG doesn't wrap these `<li>` items in `<p>` tags.  The fix here is probably to add a base font size for `<li>` to match the sizing of the `<p>` tags for accordions only.   https://engineering.uiowa.edu/bme/undergraduate#accordion-item-32046-4 is an example where the last item is wrapped in a `<p>` tag while the others are not.

**Comment by briand44** [comment]
Grooming conversation... Looks like we only addressed `<p>` previously. https://github.com/uiowa/uids/pull/974/files. Need to extend to include `<li>`. Just fix in uids like previously. Examples in https://github.com/uiowa/uiowa/issues/8505#issuecomment-2836146417

**Comment by bspeare** [comment]
I've submitted a ticket to Siteimprove.  This violation only appears when the `<details>` are closed.   Even if I add 100px of spacing, Siteimprove will still report the error.  However, if I open all the accordions and run the page there are no errors reported.
